build.gradle.kts

```
@@ -36,8 +36,9 @@ dependencies {
 	implementation("org.springframework.boot:spring-boot-starter")
 	implementation("org.springframework.boot:spring-boot-starter-web")
 	implementation("org.springframework.boot:spring-boot-starter-actuator")
-  implementation("io.micrometer:micrometer-tracing-bridge-otel:$micrometerVersion")
 	implementation("org.springdoc:springdoc-openapi-starter-webmvc-ui:$springDocOpenApiVersion")
+  	implementation("io.micrometer:micrometer-tracing-bridge-otel:$micrometerVersion")
+	implementation("io.micrometer:micrometer-registry-prometheus")
```

I see an indentation difference here
```suggestion
  implementation("io.micrometer:micrometer-tracing-bridge-otel:$micrometerVersion")
	implementation("io.micrometer:micrometer-registry-prometheus")
```